ui/lib/kv/addon/components/page/list.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { getOwner } from '@ember/application';
+
+/**
+ * @module List
+ * ListPage component is a component to show a list of kv/metadata secrets.
+ *
+ * @param {array} model - An array of models generated form kv/metadata query.
+ * @param {array} breadcrumbs - Breadcrumbs as an array of objects that contain label, route, and modelId. They are updated via the util kv-breadcrumbs to handle dynamic *secretPrefix on the list-nested-secret route.
+ */
+
+export default class KvListPageComponent extends Component {
+  @service flashMessages;
+
+  get mountPoint() {
+    return getOwner(this).mountPoint;
```

Comment:
So mountPoint is used on LinkedBlocks to determine where to transition. It's basically telling LinkedBlock where this page is located in the overall routing structure. For this component, it will always be `vault.cluster.secrets.backend.kv`. So I could hardcode it instead of returning it but the pattern seems to be calling mountPoint and passing it in.

From here LinkedBlock says transition to `vault.cluster.secrets.backend.kv.{targetRoute}`.

Reply:
Thanks for the explanation! Maybe a comment here would be nice so future us can know it's safe to remove once we replace `LinkedBlock`